1.2/percentile_test.go

```diff
@@ -60,10 +60,10 @@ func TestGenerateInvalidInput(t *testing.T) {
 	}
 }
 
-func Must(fn func(float32) (percentile, error), x float32) percentile {
-	v, err := fn(x)
+func mustGeneratePercentile(x float32) percentile {
+	v, err := generatePercentile(x)
 	if err != nil {
-		log.Fatalln(err)
+		panic("Error: percentile is nil, percentile cannot be nil")
```

Comment:
Sorry to harp on error messages.

The underlying problem isn't be that percentile is nil.  The problem is be that getPercentile returned an error.  Why did getPercentile return an error?  That information is probably in the `err` variable.  Take a look at how mustCompile constructs its error message:

https://golang.org/src/regexp/regexp.go?s=10768:10936#L298

Reply:
No, you're totally right. Brain was mush there for a sec